Library/Homebrew/utils/livecheck_formula.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module LivecheckFormula
+  module_function
+
+  def init(formula)
+    ohai "Checking livecheck formula: #{formula}" if Homebrew.args.verbose?
+
+    response = Utils.popen_read(HOMEBREW_BREW_FILE, "livecheck", formula, "--quiet").chomp
```

Comment:
CC @nandahkrishna That, once merged, this would be a good place to call into the Livecheck code rather than shell out.